versioned_docs/version-1.3.0-beta.1/how_to_guides/multiarch.mdx

```diff
@@ -0,0 +1,130 @@
+---
+title: Multiple Architecture Image Support
```

Comment:
```suggestion
title: Multi-architecture image/artifact support
```

Reply:
Maybe we can follow the common term `multi-architecture` from docker: https://www.docker.com/blog/tag/multi-architecture/

Reply:
`Multi-architecture Image / Artifact Support` <- You may want to capitalize each word.

Reply:
Done.